modules/mono/csharp_script.cpp

```diff
@@ -2016,7 +2016,7 @@ void CSharpInstance::connect_event_signals() {
 		StringName signal_name = event_signal.field->get_name();
 
 		// TODO: Use pooling for ManagedCallable instances.
-		auto event_signal_callable = memnew(EventSignalCallable(owner, &event_signal));
+		const EventSignalCallable *event_signal_callable = memnew(EventSignalCallable(owner, &event_signal));
```

Comment:
This one must not be const as `Callable`'s constructor only accepts non-const: https://github.com/godotengine/godot/blob/15a85fe9713668f8ba6143352fd27d419a96ba83/core/variant/callable.h#L105